bin/pycbc_multi_inspiral

```diff
@@ -383,21 +467,27 @@ with ctx:
                 # No clustering in the coherent search until the end.
                 # The correlation vector is the FFT of the snr (so inverse
                 # FFT it to get the snr). 
+              	# The following lines Unpack & store copies of the matched 
+              	# filtering results for the current segment and template.
                 snr_ts, norm, corr, ind, snrv = \
                      matched_filter[ifo].matched_filter_and_cluster(
                          s_num, template.sigmasq(stilde[ifo].psd), window=0)
                 snr_dict[ifo] = (
                         snr_ts[matched_filter[ifo].segments[s_num].analyze]
-                        * norm)
-                norm_dict[ifo] = norm
-                corr_dict[ifo] = corr.copy()
-                idx[ifo] = ind.copy()
-                snrv_dict[ifo] = snrv.copy()
-                snr[ifo] = snrv * norm
+                        	     * norm) # snr time series for each ifo
+                norm_dict[ifo] = norm        
+                corr_dict[ifo] = corr.copy() # The correlation vector for each
+                                             # ifo. it is the  FFT of the snr
+                                             # (so inverse FFT it to get the snr).
+                idx[ifo] = ind.copy()	     # Trigger indices list for each ifo
+                snrv_dict[ifo] = snrv.copy() # Trigger snr list for each ifo
+                snr[ifo] = snrv * norm	     # Normalized trigger snr list for each ifo
```

Comment:
Personally I find inline comments like these a bit hard to read, I suggest putting them in separate lines before the code they refer to.